spaces/domains.json

```
@@ -179,4 +179,5 @@
   "gov.parallaxprotocol.com": "parallaxprotocol.eth",
   "snapshot.everipedia.com": "everipediaiq.eth",
   "vote.nerve.fi": "nervefinance.eth"
+  "vote.mask.io": "tianranzhang.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.nerve.fi": "nervefinance.eth",
  "vote.mask.io": "tianranzhang.eth"
```